premailer/premailer.py

```diff
@@ -409,6 +411,18 @@ def transform(self, pretty_print=True, **kwargs):
                 parent = item.getparent()
                 del parent.attrib['class']
 
+        # Add align attributes to images if they have a CSS float value of
+        # right or left. Outlook (both on desktop and the web version) are
+        # bad at parsing CSS (like floats) but they do understand the HTML
+        # align attribute.
+        if self.align_floating_images:
+            for item in page.xpath("//img"):
+                image_css = cssutils.parseStyle(item.attrib.get('style', ''))
```

Comment:
This we might want to optimize a bit. 
For every single image that does NOT have a `style` attribute (not sure that there's that many) we have to go through the parse and the if statements below. How about something like this:
```python
for item in page.xpath('//img[@style]'):
    image_css = cssutils.parseStyle(item.attrib['style'])
    if ...
```